### PR TITLE
Revise default over-provisioning percentage to 100.

### PR DESCRIPTION
**Pull request by james-munson**
#7342

### CODE REVIEW

**Comment by james-munson** [comment]
Since this does not change all docs at once, like website PR https://github.com/longhorn/website/pull/816 can, it should be backported to 1.5.x but not 1.4.

File: chart/README.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ For more details like types or options, you can refer to **Settings Reference**
 | defaultSettings.snapshotDataIntegrityImmediateCheckAfterSnapshotCreation | Hashing snapshot disk files impacts the performance of the system. The immediate snapshot hashing and checking can be disabled to minimize the impact after creating a snapshot. |
 | defaultSettings.storageMinimalAvailablePercentage | If the minimum available disk capacity exceeds the actual percentage of available disk capacity, the disk becomes unschedulable until more space is freed up. By default, 25. |
 | defaultSettings.storageNetwork | Longhorn uses the storage network for in-cluster data traffic. Leave this blank to use the Kubernetes cluster network. |
-| defaultSettings.storageOverProvisioningPercentage | The over-provisioning percentage defines how much storage can be allocated relative to the hard drive's capacity. By default, 200. |
+| defaultSettings.storageOverProvisioningPercentage | The over-provisioning percentage defines how much storage can be allocated relative to the hard drive's capacity. By default, 100. |
```
Comment (jillian-maroket) [inline chart/README.md:323]:
```suggestion
| defaultSettings.storageOverProvisioningPercentage | Percentage of storage that can be allocated relative to hard drive capacity. The default value is "100". |
```

File: chart/questions.yaml [hunk 2]
```diff
@@ -299,11 +299,11 @@ questions:
     default: "disabled"
   - variable: defaultSettings.storageOverProvisioningPercentage
     label: Storage Over Provisioning Percentage
-    description: "The over-provisioning percentage defines how much storage can be allocated relative to the hard drive's capacity. By default, 200."
+    description: "The over-provisioning percentage defines how much storage can be allocated relative to the hard drive's capacity. By default, 100."
```
Comment (jillian-maroket) [inline chart/questions.yaml:302]:
```suggestion
    description: "Percentage of storage that can be allocated relative to hard drive capacity. The default value is "100"."
```

File: chart/values.yaml [hunk 3]
```diff
@@ -194,7 +194,7 @@ defaultSettings:
   replicaSoftAntiAffinity: ~
   # -- Enable this setting automatically re-balances replicas when discovered an available node.
   replicaAutoBalance: ~
-  # -- The over-provisioning percentage defines how much storage can be allocated relative to the hard drive's capacity. By default, 200.
+  # -- The over-provisioning percentage defines how much storage can be allocated relative to the hard drive's capacity. By default, 100.
```
Comment (jillian-maroket) [inline chart/values.yaml:197]:
```suggestion
# -- Percentage of storage that can be allocated relative to hard drive capacity. The default value is "100".
```

**Review by jillian-maroket — COMMENTED** [review]
@james-munson I used the [wording convention](https://suse.slack.com/archives/C02DR3N5T24/p1698308333889269) that I discussed with Jack Lin when I reviewed [PR 773](https://github.com/longhorn/website/pull/773#issuecomment-1862092462).

**Review by PhanLe1010 — APPROVED** [review]
LGTM

File: chart/README.md (hunk 1 above)
Comment (james-munson) [reply chart/README.md]: Done.

**Comment by james-munson** [comment]
Requested changes made and pushed.

**Review by jillian-maroket — APPROVED** [review]

**Review by innobead — APPROVED** [review]
LGTM. Good update!

**Comment by james-munson** [comment]
@mergify backport v1.5.x

**Pull request merged**